src/gui/mzroll/treedockwidget.cpp

```diff
@@ -285,6 +286,7 @@ void TreeDockWidget::setupScanListHeader()
 }
 
 void TreeDockWidget::addScanItem(Scan* scan) {
+        _scansList.append(scan);
```

Comment:
This should happen after the null-check in the next line.

Reply:
Added.